README.md

```diff
@@ -751,6 +751,7 @@ Projects using TOML
 - [MCPhoton](http://mcphoton.org/) - Multi-threaded Java minecraft server.
 - [MeTA](https://github.com/meta-toolkit/meta) - Modern C++ data science toolkit.
 - [Traefik](https://traefik.io/) - Modern HTTP reverse proxy.
+- [Pipenv](https://github.com/kennethreitz/pipenv) - Python's most sophisticated packaging tool (spec for Pipfile [here](https://github.com/pypa/pipfile))
```

Comment:
I think a snippet from the README of pipenv would be nicer -- "Pipenv — the officially recommended Python packaging tool from Python.org". This is a more formal description. :)

Also, I don't think pipfile needs to be linked to.